apps/wallet-mobile/src/features/Settings/SettingsScreenNavigator.tsx

```diff
@@ -138,11 +139,15 @@ const SettingsTabNavigator = () => {
         tabBarStyle: {backgroundColor: COLORS.WHITE, elevation: 0, shadowOpacity: 0},
         tabBarIndicatorStyle: {backgroundColor: COLORS.SHELLEY_BLUE, height: 2},
         tabBarLabelStyle: {
-          fontWeight: 'bold',
+          textTransform: 'none',
+          fontFamily: 'Rubik',
+          fontSize: 16,
+          fontWeight: '500',
         },
         tabBarActiveTintColor: COLORS.SHELLEY_BLUE,
         tabBarInactiveTintColor: COLORS.NOT_SELECTED_TAB_TEXT,
       })}
+      style={{marginHorizontal: 16}}
```

Comment:
To avoid this:

<img width="421" alt="Screenshot 2023-09-11 at 09 04 50" src="https://github.com/Emurgo/yoroi/assets/105349292/33f46a51-2167-4544-89c8-e3ba5dd0fa79">